### PR TITLE
Fix rendering of comparison page

### PR DESCRIPTION
**Pull request by wildermuthn**
A check already existed for a playlist returning no data. However, it returned null rather than returning the dropdowns for rank and playlist selection. 

I followed the pattern set with fragments of JSX. Ideally the conditional for rendering isn't split from the main return of JSX. But that would require some more thorough refactoring.

### CODE REVIEW

File: webapp/src/Components/Player/Compare/PlayStyle/PlayerComparePlayStyleCharts.tsx [hunk 1]
```diff
@@ -88,14 +80,39 @@ export class PlayerComparePlayStyleCharts extends React.PureComponent<Props, Sta
...
+                        handleChange={this.handleRankChange}
+                        inputLabel="Rank to compare"
+                        helperText="Select the rank to plot as average"
+                        noneLabel="Default"/>
+        )
+
+        if (playerPlayStyles.length === 0) {
+            return (
+	          <>
+                <Grid item xs={12} style={{textAlign: "center"}}>
```
Comment (twobackfromtheend) [inline webapp/src/Components/Player/Compare/PlayStyle/PlayerComparePlayStyleCharts.tsx]: innards of fragment should be indented.

Comment (twobackfromtheend) [reply webapp/src/Components/Player/Compare/PlayStyle/PlayerComparePlayStyleCharts.tsx]: I meant something like 
```
<>
    <Thing/>
</>
```

Comment (wildermuthn) [reply webapp/src/Components/Player/Compare/PlayStyle/PlayerComparePlayStyleCharts.tsx]: Yeah, the tabs were screwing things up in github. Let's see if it worked this time.

**Comment by wildermuthn** [comment]
I used a new editor that didn't strip out tabs, which made the indentation in the github viewer make it appear screwed up (default number of spaces in a tab is 4). It should be good now that I stripped them out.

**Pull request merged**